README.md

```diff
@@ -1,458 +1,478 @@
-| papr12a   | 1 | Amazon | Elastic Phone Holder | $10 | [Z][3amzn] | Elastic band to hold battery |
-| papr12b   | 1 | AliExpress | Elastic Phone Holder | $0.50 | [A][5ali] | Elastic band to hold battery |
-| papr13   | 1 | Conxwan | 26800mAh battery | $27 | [Z][4amzn] | 18W USB-C Battery |
-| papr14   | 3 | Amazon | 6-32 x 3/8" Screws | $8.75 | [Z][5amzn] | Screws to secure back |
-| papr15   | 1 | Amazon | 4ft CPAP hose | $10.57 | [Z][6amzn] | CPAP hose |
-| papr16   | 1 | 3M | 7093 Filter Cartridge (pair) | $30 | [3M][3m1] | example filter packs |
-| papr17   | about 6x6cm | McMaster-Carr | 8785K82-8785K822 (Extra Soft)  | $22.22/12x12" | [C][1mcmc] | 1/8" Silicone Foam Gasket Material |
-| papr18   | 1 | 3M | Filter Gasket  | $2.01 | [G][1grgr] | Inlet Gasket for 6000-series filters |
-| papr19   | 1 | OWO | (Optional) Upgraded potentiometer| $1.00 | [A][4ali] | nicer pot with knob and jam nut |
-| papr20   | 1 | JST | (Optional) JST connector kit and crimper | $40 | [Z][8amzn] | required connectors for upgraded potentiometer |
-
-### Mask/Helmet Parts
-Parts for modifying and adapting masks and helmets for use with the PAPR
-
-| Item | Quantity | Manufacturer  | Part #              | Price (USD)     |  Sources       | Notes |
-| ---- |---------:| ------------- | ------------------- | ------------:|----------------| ----- |
-| mask1   | 1 | 3M | 6100(S)/6200(M)/6300(L) Respirator Mask | $35 | [3M][3m2] | example half-face respirator mask* |
-| mask2   | 1 | 3M | 6700(S)/6800(M)/6900(L) Full-Face Mask | $150 | [3M][3m3] | example full-face respirator mask* |
-| mask3   | 1 | 3M | M-206 Full Face Helmet | $230 | [3M][3m4] | example full-face respiratory helmet* |
-| mask4   | 1 | Allegro | Tyvek Full-Face Hood | $23 | [Z][9amzn] | example full-face hood* |
-| mask5   | 1 | Custom 3D print | Bayonet Inlet Adapter | $2 (material) | [3D][3d7] | Inlet Adapter for Bayonet-type masks |
-| mask6   | 1 | Custom 3D print | 6100/6200/6300 Outlet Plug | $1 (material) | [3D][3d8] | Plug for sealing the outlet valve of 6100/6200/6300 masks |
-| mask7   | 1 | 3M | Inlet Filter Gasket  | $2.01 | [G][1grgr] | Inlet Gasket used as part of above outlet sealing |
+# RespiraWorks PAPR Project
+
+![Overview photo](/assets/Overview-PAPR-Collage.JPG)
+
+***Important Disclaimer: This is a prototype device, offered free-of-charge as a public service during the COVID emergency, with no claim, guarantee, or warranty offered or implied of performance or suitablity for any purpose.  This device has not be independently certified or approved.  Use of this design or any derivatives thereof is entirely at your own risk.  By using any of these materials, you acknowledge that you have read and understood the [full disclaimer](#full-disclaimer) at the end of this document.  Please do not use these materials if you do not understand and accept these risks.***

```

Comment:
"has not be independently" should be "has not been independently"

Reply:
Thanks for catching that, fixed now.